cluster-autoscaler/utils/test/test_utils.go

```diff
@@ -240,6 +241,24 @@ func GetGPULabel() string {
 	return gpuLabel
 }
 
+// GetAcceleratorFromNode returns the accelerator of the node if it has one. This is only used in unit tests.
+func GetAcceleratorFromNode(node *apiv1.Node) cloudprovider.Accelerator {
+	gpuType, hasGpuLabel := node.Labels[gpuLabel]
+	gpuAllocatable, hasGpuAllocatable := node.Status.Allocatable[resourceNvidiaGPU]
+	if hasGpuLabel || (hasGpuAllocatable && !gpuAllocatable.IsZero()) {
+		return cloudprovider.Accelerator{
+			Label:        gpuLabel,
+			Type:         gpuType,
+			ResourceName: resourceNvidiaGPU,
+		}
+	}
+	return cloudprovider.Accelerator{
+		Label:        "",
+		Type:         "",
+		ResourceName: "",
+	}
```

Comment:
This is equivalent to `return cloudprovider.Accelerator{}`, but much less concise.

Reply:
Now returning nil, instead of an accelerator with empty strings.